scripts/release/publish.js

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+
```

Comment:
Maybe I missed it, but could we add a prompt after the 'release' to create the 'release' on Github? This is a separate step done via the Github UI.

Reply:
That's part of the printed instructions once this script completes (in `print-post-publish-summary`):

> **Step 1: Create GitHub release**
> 1. Open new release page: **https://github.com/facebook/react/releases/new**
> 2. Choose **${version}** from the dropdown menu
> 3. Paste the new release notes from **CHANGELOG.md**
> 4. Attach all files in **build/dist/*.js** except **react-art.*** to the release.
> 5. Press **"Publish release"**!

Reply:
Are you planning on automating the GitHub release creation in the future too? We did this for Yarn 😄

Reply:
Sure. Sounds awesome. We can just copy what you've already done. 👍